### PR TITLE
updated README.md and added swagger.yaml

### PR DESCRIPTION
**Pull request by anwilkie**
# Motivation and Context
Like the other services, the collection instrument service required swagger documentation.

# What has changed

- Added `swagger.yaml` file
- Added service information and improvement suggestions to `README.md`
- Removed `API.md`

# Links
[Trello card](https://trello.com/c/hg1Q7l06)

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+          required: true
+          schema:
+            type: string
+            format: dictionary
+            example: '{"FORM_TYPE":%20"001"}'
+          description: Collection instrument classifiers
+      responses:
+        '200':
+          description: Successfully uploaded collection instrument
+          content: "The upload was successful"
```
Comment (jterryons) [inline swagger.yaml:96]: This is not quite right. Whats the content type? `application/json`? `text/plain`? If its JSON you need to supply a schema. its not enought to just have `content: "some string"`
look at the API.yaml in the other repos for examples.

File: swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+          schema:
+            type: string
+            example: 'file_name.xlsx'
+          description: Spreadsheet file
+        - in: query
+          name: classifiers
+          required: true
+          schema:
+            type: string
+            format: dictionary
```
Comment (jterryons) [inline swagger.yaml:90]: are we really inputting a raw JSON string as the `classifiers` query param?

File: swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+info:
+  title: collection instrument
+  description: A service for the uploading of collection instruments and exercises
+  version: "2.0"
+
+servers:
+  - url: https://localhost:8080/
+    description: The path used by the collection exercise service when port forwarded by kubectl (8080:8080)
+
+tags:
```
Comment (jterryons) [inline swagger.yaml:11]: this is a little tag heavy. Get rid of all tags other than info and collection-instrument for now.

File: swagger.yaml (hunk 1 above)
Comment (anwilkie) [reply swagger.yaml]: I tried looking at the other API.yaml files, but I couldn't find any endpoints where only a message is returned. I have modified these `content:` tags to `text/plain` objects.

File: swagger.yaml (hunk 2 above)
Comment (anwilkie) [reply swagger.yaml]: Sorry, this was an error caused by me sloppily copying and pasting stuff around. I've deleted this now.

File: swagger.yaml (hunk 3 above)
Comment (anwilkie) [reply swagger.yaml]: I've reduced the tags to just `collection-instrument`, `info` and `survey-response`.

**Pull request merged**